src/api/FileVersionAPI.js

```diff
@@ -91,7 +91,7 @@ class FileVersionAPI extends API {
                 marker: next_marker
             };
 
-            this.fetchFromMarker(params);
+            return this.fetchFromMarker(params);
         }
 
         if (data.type === 'error' || !Array.isArray(data.entries)) {
```

Comment:
I didn't realize this until now, but how come we consider an error a part of a successful annotations fetch?

Reply:
Went WAY back but don't really have a solid reasoning behind why this is there :-/